lib/foreman_tasks/dynflow.rb

```diff
@@ -43,14 +43,17 @@ def initialize!
         unless config.remote?
           at_exit { world.terminate.wait }
 
-          # for now, we can check the consistency only when there
-          # is no remote executor. We should be able to check the consistency
-          # every time the new world is created when there is a register
-          # of executors
-          world.consistency_check
-          world.execute_planned_execution_plans
-
-          Task::DynflowTask.consistency_check
+          # don't try to do any rescuing until the tables are properly migrated
+          if !Foreman.in_rake?('db:migrate') && (ForemanTasks::Task.table_exists? rescue(false))
+            # for now, we can check the consistency only when there
+            # is no remote executor. We should be able to check the consistency
+            # every time the new world is created when there is a register
+            # of executors
+            world.consistency_check
```

Comment:
Yes, there was a change on dynflow 0.8.0, when testing this PR with the 0.7.0 one, you need to update these lines the same way as https://github.com/theforeman/foreman-tasks/pull/116/files#diff-c48e58b017e4345d277b2cbf51e1b159L50, replacing the `consistency_check` and `execute_planned_exection_plans` with `auto_execute`